### DISCUSSION TITLE
How to get smart table item with ts-sdk?

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#194] [Questions] [answered]
### Discord user ID

ice1118761

### Describe your question in detail.

As title. Asked in discord dev-discussion.

### What have you tried or looked at?  Or how can we reproduce the error?

it seems the current function aptos.getTableItem only works for the normal table in move

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by magnum6actual** [comment, marked as answer]
There isn't a simple way in the SDK right now. The easiest path will be to write your own view function in the module that owns the smart table resource you're trying to access.
